extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html

```diff
@@ -7,18 +7,18 @@
 					<div class="parameter col-sm-6 col-xs-12" ng-switch="parameter.element">
 						<div ng-switch-when="input">
 							<div ng-if="parameter.type=='INTEGER'">
-								<md-input-container> <label>{{parameter.label}}</label> <input ng-init="focus=false" name="{{parameter.name}}" type="number" min="{{parameter.min}}" max="{{parameter.max}}" ng-model="configuration[parameter.name]" ng-required="parameter.required" type="{{parameter.inputType}}" ng-focus="focus=true" ng-blur="focus=false">
-								<div ng-messages="form.configForm.{{parameter.name}}.$error" ng-show="focus">
-									<div ng-message="min">Minimum allowed number is {{parameter.min}}</div>
-									<div ng-message="max">Maximum allowed number is {{parameter.max}}</div>
-									<div ng-message="required">Field is required</div>
-								</div>
-								<span ng-messages="" ng-show="!focus">
-									<p class="hint">
-										<small>{{parameter.description}}</small>
-									</p>
-								</span> </md-input-container>
-							</div>
+                                <md-input-container> <label>{{parameter.label}}</label> <input ng-init="focus=false" name="{{parameter.name}}" type="number" min="{{parameter.min}}" max="{{parameter.max}}" ng-model="configuration[parameter.name]" ng-required="parameter.required" type="{{parameter.inputType}}" ng-focus="focus=true" ng-blur="focus=false">
```

Comment:
something is messed up with the formatting here - the html files should have tabs.